iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/parameters/ConversionParameters.java

```diff
@@ -78,14 +76,22 @@ public static double readDoubleParameter(String format, Properties parameters, P
     }
 
     private static <T> T readParameter(String format, Properties parameters, Parameter configuredParameter, T defaultValue,
```

Comment:
@miovd @flo-dup all static methods of ConversionParameters could go in Parameters  and ConversionParameters should better be removed?

Reply:
Done!